tests/Constraints/CoerciveTest.php

```diff
@@ -128,7 +128,7 @@ public function getValidCoerceTests()
                   "boolean":"true",
                   "object":{},
                   "array":[],
-                  "null":null,
+                  "null":"",
```

Comment:
I notice that you have lots of explicit tests in `testValidCoerceCases`, in addition to simply requiring the schema to validate, but that null coercion hasn't been added as one of them. Is there a reason for that?